conf/domain_mix/balanced.yaml

```diff
@@ -0,0 +1,9 @@
+# @package actor.domain_mix
+
+math: 1.0
```

Comment:
Are these names coming from the `environments[n].key` field?

Reply:
no, the keys in actor.domain_mix must match the domain field on the samples used for training. the sampler expects domain labels per sample